<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceDownloader</class>
 <widget class="QWidget" name="SequenceDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequence Downloader</string>
  </property>
  <widget class="QPushButton" name="loadSequenceButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Sequence Frames</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the sequence to download,
remember to unzip it afterwards.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>209</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="sequenceRadio_01">
      <property name="text">
       <string>01 - Water bottle</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sequenceRadio_02">
      <property name="text">
       <string>02 - Bowling pin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sequenceRadio_03">
      <property name="text">
       <string>03 - Small bottle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sequenceRadio_04">
      <property name="text">
       <string>04 - Sphere</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
